10-inference.Rmd

```diff
@@ -0,0 +1,789 @@
+might expect our point estimate to behave if we took another sample.
+
+```{r 11-bootstrapping5, echo = TRUE, message = FALSE, warning = FALSE, fig.cap = "Distribution of the bootstrap sample means"}
+boot_est_dist <-  ggplot(boot1500_means, aes(x = mean)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef") +
+    xlab("Sample mean price per night ($)") 
+```
+
+Let's compare our bootstrap distribution with the true sampling distribution (taking many samples from the population). 
+
+```{r 11-bootstrapping6, echo = F, message = FALSE, warning = FALSE, fig.cap = "Comparison of distribution of the bootstrap sample means and sampling distribution"}
+
+samples <- rep_sample_n(airbnb, size = 40, reps = 1500)
+
+sample_estimates <- samples %>% 
+    group_by(replicate) %>% 
+    summarise(sample_mean = mean(price))
+
+sampling_dist <-  ggplot(sample_estimates, aes(x = sample_mean)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef") +
+    xlab("Sample mean price per night ($)")
+
+sampling_dist <- sampling_dist +
+  annotate("text", x = max_x(sampling_dist), max_count(sampling_dist), hjust = 1, vjust =1,
+           label = paste("mean = ", round(mean(sample_estimates$sample_mean), 1))) + 
+  annotate("text", x = max_x(sampling_dist), max_count(sampling_dist), hjust = 1, vjust =3,
+           label = paste("sd = ", round(sd(sample_estimates$sample_mean), 1))) + 
+  geom_vline(xintercept = mean(sample_estimates$sample_mean), col = "red")
+
+annotated_boot_est_dist <- boot_est_dist + 
+  annotate("text", x = max_x(boot_est_dist), max_count(boot_est_dist), hjust = 1, vjust =1,
+           label = paste("mean = ", round(mean(boot1500_means$mean), 1))) + 
+    annotate("text", x = max_x(boot_est_dist), max_count(boot_est_dist), hjust = 1, vjust =3,
+           label = paste("sd = ", round(sd(boot1500_means$mean), 1))) + 
+  geom_vline(xintercept = mean(boot1500_means$mean), col = "red")
+
+grid.arrange(sampling_dist + xlim(min_x(sampling_dist), max_x(sampling_dist)) + ggtitle("Sampling distribution"), 
+             annotated_boot_est_dist + xlim(min_x(sampling_dist), max_x(boot_est_dist)) + ggtitle("Bootstrap distribution"),
+             ncol = 2)
+```
+There are two important points that we can take away from these plots.
+First, we see how the sampling distribution is centred at \$`r round(mean(airbnb$price),2)`, the 
+population mean value, as we learned in
+the previous section. However, the bootstrap distribution is centred at \$`r round(mean(boot1500_means$mean), 2)`, which 
+is the original sample's mean price per night. This is because we are resampling from the original sample over and
+over again, so we see that the bootstrap distribution is centred at the
+original sample's mean value. The second important point is that the shape and spread of the true sampling distribution
+and the bootstrap distribution is similar; the bootstrap distribution lets us get a sense for the variability of the 
+point estimate.
```

Comment:
The most important point here is that the bootstrap distribution pretty well approximates the spread of the sampling distribution. And we can use that spread to help come up with a plausible range for where we expect our population parameter's value to lie. 

A secondary note - is that unlike the sampling distribution's mean, the bootstrap distribution's mean is not the value of the population parameter - this is because all the bootstrap samples came from a single sample from the population - and so most of their means will be similar to that single sample (so the bootstrap distribution's mean, if we did enough replicates would converge on the mean of the single sample).

Here's what I used in my MDS notes this year:

What do we notice?

The shape and spread of the distribution of the bootstrap sample means approximates that of the sampling distribution of sample means!

The means of these two distributions are different: the mean of the boostrap sample means is almost exactly that of the original one sample mean, whereas the mean of the sampling distribution of sample means is almost exactly that of the population parameter we are interested in estimating.

Idea! We can use this distribution of the bootstrap sample means to approximate the sampling distribution of sample means when we only have one sample - and then we can use this to generate a plausible range where we believe the value of true population parameter lies to report along with our estimate!